### ISSUE TITLE
Where is the bbox info when train ?

### ISSUE THREAD

**Issue by omg777** [marvis/pytorch-yolo2#108]
If i run this code 
`python` train.py cfg/voc.data cfg/yolo-voc.cfg darknet19_448.conv.23`

where is the bounding box information (.xml files) ?

cfg/voc.data  : image list and class list.
cfg/yolo-voc.cfg : model.

**Comment by daixiaoxi** [comment]
Hello, the voc.data contains the info of the train and vaild, the net can get the (img,label) from the train_loader and the test_loader in the train.py.
